lib/src/provider/asset_picker_provider.dart

```diff
@@ -56,13 +56,25 @@ abstract class AssetPickerProvider<Asset, Path> extends ChangeNotifier {
   /// 销毁时重置所有内容
   @override
   void dispose() {
+    _mounted = false;
```

Comment:
Maybe put this line before `super.dispose` to match its semantics.

Reply:
sure